clients/drcachesim/tests/view_test.cpp

```diff
@@ -586,11 +586,74 @@ run_chunk_tests(void *drcontext)
     return run_single_thread_chunk_test(drcontext) && run_serial_chunk_test(drcontext);
```

Comment:
> Specifically, when we disassemble DR_ISA_REGDEPS instructions, we print the
instruction encoding (can span multiple lines), we substitute the opcode with
the categories, we print the operations size (e.g., [4byte]), and then the
virtual register names (e.g., %rv3).

This sounds like this PR is implementing all of that, but it's not.  Please reword to make it clear that's coming from DR's disassembler and is not added by this PR.

Reply:
Done.